src/Nethermind/Nethermind.Api/NethermindApi.cs

```diff
@@ -100,6 +101,8 @@ public IBlockchainBridge CreateBlockchainBridge()
             );
         }
 
+        public IBlockProcessorFactory BlockProcessorFactory { get; set; }
```

Comment:
I would like to separate the factories from the rest of the APIs.
something like this
```
NethermindApi.Factories.BlockProcessingFactory
```